sql/hive/src/test/scala/org/apache/spark/sql/sources/SimpleTextRelation.scala

```diff
@@ -156,6 +156,7 @@ class CommitFailureTestRelation(
         context: TaskAttemptContext): OutputWriter = {
       new SimpleTextOutputWriter(path, context) {
         override def close(): Unit = {
+          super.close()
```

Comment:
I was thinking about S3, where a file is not actually created before the output stream is closed (the `PUT` operation happens in `NativeS3FsOutputStream.close()`). But `SimpleTextRelation` is only used for local testing, so yeah, this line is not necessary.